tests/src/analysis/testqgsprocessing.cpp

```diff
@@ -2508,12 +2528,23 @@ void TestQgsProcessing::parameterPoint()
   QGSCOMPARENEAR( point.x(), 122451, 100 );
   QGSCOMPARENEAR( point.y(), 244963, 100 );
 
+  params.insert( "non_optional", QString( "  ( 1.1,2.2   [EPSG:4326]   ) " ) );
+  QCOMPARE( QgsProcessingParameters::parameterAsPointCrs( def.get(), params, context ).authid(), QStringLiteral( "EPSG:4326" ) );
+  point = QgsProcessingParameters::parameterAsPoint( def.get(), params, context, QgsCoordinateReferenceSystem( "EPSG:3785" ) );
+  QGSCOMPARENEAR( point.x(), 122451, 100 );
+  QGSCOMPARENEAR( point.y(), 244963, 100 );
+
   // nonsense string
   params.insert( "non_optional", QString( "i'm not a crs, and nothing you can do will make me one" ) );
   point = QgsProcessingParameters::parameterAsPoint( def.get(), params, context );
   QCOMPARE( point.x(), 0.0 );
   QCOMPARE( point.y(), 0.0 );
 
+  params.insert( "non_optional", QString( "   (   )  " ) );
```

Comment:
`s/QString/QStringLiteral` :stuck_out_tongue_closed_eyes:

Reply:
I'll do that in another PR - this file is lit up like a christmas tree by qtcreator's clazy checks for that same reason.

Reply:
Guideline to interpret this comment

> If that's the only thing he complains about and if it's inside a test source file, maybe I should just click this merge button.